src/basic/path-util.h

```diff
@@ -42,9 +42,15 @@
 #  define DEFAULT_USER_PATH DEFAULT_PATH
 #endif
 
-bool is_path(const char *p) _pure_;
+static inline bool is_path(const char *p) {
+        return strchr(p, '/');
+}
+
+static inline bool path_is_absolute(const char *p) {
+        return p[0] == '/';
+}
+
```

Comment:
one would hope tlo actually figures this out on its own... but yeah, probably should be inline

maybe while you are at it, also check if p is non-NULL, after all a NULL ptr is not a path and not an absolute path either. might make things a bit more robust, dunno